loolwsd.xml.in

```diff
@@ -170,7 +170,7 @@
     </welcome>
 
     <user_interface>
-      <mode type="string" desc="Controls the user interface style (classic|notebookbar)" default="@USER_INTERFACE_MODE@">@USER_INTERFACE_MODE@</mode>
+      <mode type="string" desc="Controls the user interface style (classic|notebookbar)" default="classic">@USER_INTERFACE_MODE@</mode>
```

Comment:
Shouldn't we put "notebookbar" here, to indicate the default?

Reply:
Yes, we should put "notebookbar" to defaut and instead of @USER_INTERFACE_MODE@, too.
1. Old configs without user_interface.mode will get the new default: **notebookbar**.
2. Old configs with user_interface.mode will get what is set there.
3. New installations will get the **notebookbar** by default. I don't see other cases.